crd-docs/crd/kiali.io_kialis.yaml

```
@@ -1297,9 +1297,18 @@ spec:
                             description: "Specific properties when the collector type is `otel`."
                             type: object
                             properties:
+                              ca_name:
+                                description: "The name of the ca name containing when `tls_enabled` is `true` and `skip_verify` is `false`."
+                                type: string
                               protocol:
                                 description: "Protocol. Supported values are: `http`, `https` or `grpc`."
                                 type: string
+                              skip_verify:
+                                description: "Skip tls verification when using self signed certificates. Insecure option recommended just for testing."
+                                type: boolean
+                              tls_enabled:
+                                description: "Enable tls for the collector. It must be used when `protocol` is `https` or `grpc`. It needs to set `ca_name` or `skip_verify`."
```

Capitalize TLS and tell the reader it "must be *specified*".  Also, be more clear about what else is required.

```suggestion
                                description: "Enable TLS for the collector. This must be specified when `protocol` is `https` or `grpc`. When you set this to `true`, you must also set a `ca_name` or set `skip_verify` to true."
```